src/schedule/kat_key_schedule.rs

```diff
@@ -108,76 +109,81 @@ fn generate(
     )
 }
 
+#[cfg(any(feature = "expose-test-vectors", test))]
```

Comment:
Since the entire module is only included if `#[cfg(test)]` these feature configs don't make any difference.

Reply:
Right, module exposure is something I missed in this PR and have "fixed" in the interop branch. But as I said in another comment, after the interop week is over now, I need to take some time to figure out how to do these things cleanly.